internal/models/users/user_ranking/user_ranking.go

```diff
@@ -91,14 +77,29 @@ ORDER BY total_points DESC;
 	return rankings, nil
 }
 
-// Returns the ranking of the specified user.
-func GetUserRanking(db *sql.DB, userID uuid.UUID, month time.Month, year int, timeZone *time.Location) (UserRanking, error) {
+// Returns the first and last moment of the specified date range.
+func GetDateRange(dateRange DateRange, timeZone *time.Location, year int, month time.Month) (time.Time, time.Time) {
```

Comment:
if i pass in dateRange `All`, both year and month parameters will be ignored right? same with `Year`, then month will be ignored i assume.  
Not sure if there is an alternative way of doing it tho

Reply:
No idea actually

Reply:
💀